tests/pass/simple_py3.py

```diff
@@ -0,0 +1,6 @@
+import numpy as np
+
+array = np.array([1, 2])
+
+# The @ operator is not in python 2
+array @ array
```

Comment:
Some people on 3.4 may want to run the test suite, so perhaps this shouldn't be used? Mypy defaults to using the python version of the executable used to run it.

Reply:
We could try to come up with a more generalized way of specifying what versions of python a given file can run on, or extend the mechanism so we have something like `_py<min_version>.py` files, where `min_version` could be 3, but could also be something like 3.5. If you think that's worth it I'm happy to do that :)